src/Service/DataProvider.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Service;
+
+class DataProvider
+{
+    public function getStudents(): array
+    {
+        $students = [];
+        $storage = json_decode($this->getStorage(), true);
+        foreach ($storage as $teams) {
+            foreach ($teams['students'] as $student) {
+                $students[] = strtolower($student);
+            }
+        }
+
+        return $students;
+    }
+
+    public function getTeams(): array
+    {
+        return array_keys(json_decode($this->getStorage(), true));
```

Comment:
Šiuo atveju pavyzdiniai duomenys yra mažosiomis raidėmis.
Bet yra gera praktika:
jei norima tikrinti nepriklausomai nuo didžiųjų/mažųjų raidžių – reikėtų pakeiti abi puses:
```suggestion
        return array_map('strtolower', array_keys(json_decode($this->getStorage(), true)));
```
Nes kitoje pusėje tikrinama su `strtolower`:
https://github.com/aurelijusb/kickstart/blob/440fc3769faffcd7c76bd5d23f3e483b23485949/src/Controller/PeopleController.php#L50

Reply:
Taip, su studentais pridėjau, o čia ir pasimetė.